src/components/Button/Button.jsx

```diff
@@ -1,11 +1,20 @@
 import "./Button.scss";
 
-const Button = ({ buttonText, buttonName, onClick }) => {
+const Button = ({
+  buttonText,
+  buttonName,
+  onClick,
+  buttonImgUrl,
+  buttonImgAlt,
+  buttonImgClass,
+}) => {
   let buttonClass = `button button--${buttonName}`;
+  let buttonImageName = `button--image--${buttonImgClass}`;
 
   return (
     <button className={buttonClass} onClick={onClick}>
       {buttonText}
+      <img className={buttonImageName} src={buttonImgUrl} alt={buttonImgAlt} />
```

Comment:
Not every button needs an `<img>`, so perhaps hide this if there is no `buttonImgUrl`?